openapi/geonames/Package.md

```diff
@@ -0,0 +1,17 @@
+Connects to [Geonames API](https://www.geonames.org/export/JSON-webservices.html) from Ballerina
+
+## Package overview
+The `ballerinax/geonames` is a [Ballerina](https://ballerina.io/) connector for Flat platform. This package provides the capability to retrieve information about geo locations. 
+
+### Compatibility
+|                    | Version          |
+|--------------------|------------------|
+| Ballerina Language |  Swan Lake Beta3 |
```

Comment:
Missing API version

Reply:
Fixed in https://github.com/ballerina-platform/openapi-connectors/commit/45c0ec66caf0d3ead895a90f7f2ff2ef6c431654